### DISCUSSION TITLE
Validate option in examples

### DISCUSSION THREAD

**Discussion by jessekrubin** [MichalLytek/typegraphql-prisma#310] [Q&A]
Hi Michal!

Big fan of this library. I noticed in your examples you mostly have 'validate: false`, why is that? and what does that do? and is it reccomended?

Thanks,
Jesse

**Comment by MichalLytek** [comment]
Have you read TypeGraphQL docs?

**Reply by jessekrubin** [reply]
Yes I have. I am wondering why you use it with `typegraphql-prisma` specifically?

**Reply by MichalLytek** [reply]
I'm quite sure that's because examples in `typegraphql-prisma` do not use `class-validator` decorators 🤔
